### ISSUE TITLE
install stalled

### ISSUE THREAD

**Issue by ejb1123** [Antergos/Cnchi#268]
the install gets stuck at 
`Installing using the 'automatic' method. Please wait...`
I am using  version 0.6.49

**Comment by karasu** [comment]
Hi @ejb1123 ,

Could you rerun the installer with debug logging enabled?

Just run:

sudo -E cnchi -dv

It will create a log in /tmp/cnchi.log, just post it here.

Thanks!

**Comment by ejb1123** [comment]
```
[2015-03-06 08:28:44] [mainwindow.py] INFO: Cnchi installer version 0.6.49
[2015-03-06 08:28:50] [timezone.py] INFO: We have connection. Let's get our timezone
[2015-03-06 08:28:50] [timezone.py] INFO: Timezone detected.
[2015-03-06 08:28:53] [check.py] INFO: We have Internet connection.
[2015-03-06 08:28:53] [check.py] INFO: We're connected to a power source.
[2015-03-06 08:28:53] [check.py] INFO: We have enough disk space.
[2015-03-06 08:28:53] [connectionpool.py] INFO: Starting new HTTP connection (1): www.archlinux.org
[2015-03-06 08:28:53] [connectionpool.py] DEBUG: "GET /mirrors/status/json/ HTTP/1.1" 301 366
[2015-03-06 08:28:53] [connectionpool.py] INFO: Starting new HTTPS connection (1): www.archlinux.org
[2015-03-06 08:28:53] [connectionpool.py] DEBUG: "GET /mirrors/status/json/ HTTP/1.1" 200 17638
[2015-03-06 08:28:53] [rank_mirrors.py] DEBUG: Auto mirror selection has been run successfully
[2015-03-06 08:29:00] [location.py] INFO: locale changed to : en_US.UTF-8
[2015-03-06 07:29:00] [timezone.py] INFO: location changed to : America/Chicago
[2015-03-06 07:29:01] [timezone.py] INFO: location changed to : America/Chicago
[2015-03-06 07:29:07] [location.py] INFO: locale changed to : en_US.UTF-8
[2015-03-06 07:29:07] [timezone.py] INFO: location changed to : America/Chicago
[2015-03-06 08:29:09] [timezone.py] INFO: location changed to : America/New_York
[2015-03-06 08:29:09] [timezone.py] INFO: location changed to : America/New_York
[2015-03-06 08:29:11] [keymap.py] INFO: keyboard_layout is USA
[2015-03-06 08:29:13] [desktop.py] INFO: Cnchi will install Antergos with the 'gnome' desktop
[2015-03-06 08:29:13] [features.py] DEBUG: Detected bluetooth device. Enabling by default...
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'aur' feature to install
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'bluetooth' feature to install
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'cups' feature to install
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'fonts' feature to install
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'office' feature to install
[2015-03-06 08:29:17] [features.py] DEBUG: Selected 'firewall' feature to install
[2015-03-06 08:29:21] [automatic.py] DEBUG: ATA WDC WD10JPCX-24U [1000 GB] (/dev/sda)
[2015-03-06 08:29:21] [automatic.py] DEBUG: Lexar USB Flash Drive [8 GB] (/dev/sdb)
[2015-03-06 08:29:24] [automatic.py] INFO: Automatic install on /dev/sda
[2015-03-06 08:29:24] [automatic.py] INFO: Cnchi will install Antergos on /dev/sda
[2015-03-06 08:29:24] [automatic.py] INFO: Antergos will install the GRUB2 bootloader on /dev/sda
[2015-03-06 08:29:24] [process.py] DEBUG: Creating partitions and their filesystems in /dev/sda
[2015-03-06 08:29:24] [auto_partition.py] DEBUG: Total disk size: 953869MiB
[2015-03-06 08:29:24] [auto_partition.py] DEBUG: Boot partition size: 256MiB
[2015-03-06 08:29:24] [auto_partition.py] DEBUG: Swap partition size: 5889MiB
[2015-03-06 08:29:24] [auto_partition.py] DEBUG: Root partition size: 947723MiB
[2015-03-06 08:29:48] [slides.py] INFO: Installing using the 'automatic' method. Please wait...
```

**Comment by karasu** [comment]
Thanks, but it's odd as I don't see any errors ¿?
It seems the auto partition process stops for no apparent reason.
I've just tested now in a VM and works for me.

Are you using an UEFI system? Have you tried the advanced installation?

**Comment by ejb1123** [comment]
The machine is a UEFI. I am installing using non UEFI boot from the flash drive. I just did the advance install and from the previous install attempt it made a fat32 efi partition, a swap partition, and what would be "/", with a question mark. Going through the advance installer fixed the issue. I fixed the issue by making the partition table GPT, and making the correct partitions. The label for "efi /boot" was not at the bottom of the installer. My guess is that the simplified installer guessed I was installing for a UEFI Boot.
